lib/config-validator.ts

```diff
@@ -75,7 +76,7 @@ type PackageJson = {
         const parsedContent = await getParsedContent(file);
         try {
           logger.info(`Validating ${file}`);
-          await validate(file, parsedContent, strict);
+          await validate(false, file, parsedContent, strict);
```

Comment:
```suggestion
          await validate(true, file, parsedContent, strict);
```
We can fix the breaking change by allowing global options in the config, if the user is validating a custom file by default.

In another PR we can add a flag to control this behaviour.